ntcore/src/test/native/cpp/EntryNotifierTest.cpp

```diff
@@ -12,10 +12,10 @@
 #include "ValueMatcher.h"
 #include "gtest/gtest.h"
 
+using ::testing::_;
```

Comment:
wpiformat bug?

Reply:
clang-format is what sorts those. What version are you using?